src/PactNet.Native/NativeRequestBuilder.cs

```diff
@@ -163,6 +173,15 @@ IRequestBuilderV3 IRequestBuilderV3.WithQuery(string key, string value)
         IRequestBuilderV3 IRequestBuilderV3.WithHeader(string key, string value)
             => this.WithHeader(key, value);
 
+        /// <summary>
+        /// Add a request header
+        /// </summary>
+        /// <param name="key">Header key</param>
+        /// <param name="valueMatcher">Header value matcher</param>
+        /// <returns>Fluent builder</returns>
+        IRequestBuilderV3 IRequestBuilderV3.WithHeader(string key, IMatcher valueMatcher)
+            => this.WithHeader(key, valueMatcher);
```

Comment:
I'm gonna disagree with @yanncourtel a bit here I think. I think that var name is quite clunky. I would've just called it matcher.

Reply:
My point was consistency across the same type of methods. One of the variable was called valueBuilder which I found it odd (prob copy/paste thingy). You are right, matcher is actually a better name.